hera_mc/cm_hookup.py

```diff
@@ -136,7 +136,9 @@ def __follow_hookup_stream(self, part, rev, port, sn):
     def get_hookup(self, hpn, rev, port, at_date, state_args, exact_match=False):
         """
         Return the full hookup.
-        Returns hookup_dict, a dictionary keyed on derived key of hpn:port.
+        Returns hookup_dict, a dictionary with two entries:
+            'hookup': another dictionary keyed on part:rev:port:sn
+            'columns': names of column headers
```

Comment:
What does 'column headers' mean? It seems to refer to something in a table, but I don't know what that table is.